Source/Scene/Cesium3DTileset.js

```diff
@@ -1995,14 +1995,19 @@ function makeTile(tileset, baseResource, tileHeader, parentTile) {
     const contentUri = implicitTileset.subtreeUriTemplate.getDerivedResource({
       templateValues: rootCoordinates.getTemplateValues(),
     }).url;
-    const contentJson = {
-      content: {
+
+    const deepCopy = true;
+    const tileJson = clone(tileHeader, deepCopy);
+    // Replace contents with the subtree
+    tileJson.contents = [
+      {
         uri: contentUri,
       },
-    };
+    ];
 
-    const deepCopy = true;
-    const tileJson = combine(contentJson, tileHeader, deepCopy);
+    if (defined(tileJson.content)) {
+      delete tileJson.content;
+    }
```

Comment:
`delete` always works even if the key is not defined, the if clause can be removed.